app/src/main/java/pm/gnosis/heimdall/ui/safe/recover/phrase/RecoverInputRecoveryPhraseActivity.kt

```diff
@@ -3,22 +3,25 @@ package pm.gnosis.heimdall.ui.safe.recover.phrase
 import android.content.Context
 import android.content.Intent
 import kotlinx.android.synthetic.main.layout_input_recovery_phrase.*
+import pm.gnosis.heimdall.R
 import pm.gnosis.heimdall.di.components.ViewComponent
 import pm.gnosis.heimdall.reporting.ScreenId
 import pm.gnosis.heimdall.ui.safe.main.SafeMainActivity
 import pm.gnosis.heimdall.ui.safe.mnemonic.InputRecoveryPhraseActivity
+import pm.gnosis.heimdall.ui.safe.mnemonic.InputRecoveryPhraseContract
 import pm.gnosis.model.Solidity
-import pm.gnosis.svalinn.common.utils.*
+import pm.gnosis.svalinn.common.utils.toast
+import pm.gnosis.svalinn.common.utils.visible
 
 class RecoverInputRecoveryPhraseActivity : InputRecoveryPhraseActivity<RecoverInputRecoveryPhraseContract>() {
 
-    override fun noRecoveryNecessary(safe: Solidity.Address) = onSuccess(safe)
+    override fun noRecoveryNecessary(safe: Solidity.Address) = toast(R.string.no_recovery_necessary)
```

Comment:
Why are we changing this?

`noRecoveryNecessary` just indicates that we can directly add the safe to the app since it is already in the requested configuration. So we should just add it (what it did before)